frontend/components/AddBookForm/AddBookForm.js

```diff
@@ -29,112 +29,109 @@ const AddBookForm = ({ onClick }) => {
 
   const findAuthor = (fName, lName) => {
     return authors.find(
-      (author) => author.firstName === fName && author.lastName === lName
+      author => author.firstName === fName && author.lastName === lName
     );
   };
 
-  const submitForm = async (data) => {
+  const submitForm = async data => {
     const title = data.Title;
     const description = data.Description;
-    const fName = data["First Name"];
-    const lName = data["Last Name"];
+    const fName = data['First Name'];
+    const lName = data['Last Name'];
```

Comment:
Maybe there is a better way we can:
1) name these data variables, so they can be accessed in dot notation
2) destructure them in a way that can compress this line into a few less.

Reply:
const {
      First_Name: fName,
      Last_Name: lName,
      Title: title,
      Description: description
    } = data;
    
    This was the solution I came up with. I have a converter function that converts the label from First Name to First_Name when the SingLine text component is created.